R/pkg/R/mllib.R

```diff
@@ -41,7 +41,8 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' sqlContext <- sparkRSQL.init(sc)
 #' data(iris)
 #' df <- createDataFrame(sqlContext, iris)
-#' model <- glm(Sepal_Length ~ Sepal_Width, df)
+#' model <- glm(Sepal_Length ~ Sepal_Width, df, family=c("gaussian","binomial"))
```

Comment:
And if we specify a list it is taking the first one: 
 family=c("gaussian","binomial")
 summary(model)
$coefficients
              Estimate
(Intercept)  6.5262226
Sepal_Width -0.2233611

family = "gaussian"
summary(model)
$coefficients
              Estimate
(Intercept)  6.5262226
Sepal_Width -0.2233611